CHANGELOG.md

```diff
@@ -1,56 +1,303 @@
+consume the credentials output by `tbot` .
+
+For more information, see [docs](docs/pages/machine-id/reference/configuration.mdx#kubernetes_secret).
+
+### Breaking changes and deprecations
+
+Please familiarize yourself with the following potentially disruptive changes in
+Teleport 14 before upgrading.
 
-If, after applying user traits, only one non-wildcard db name/user is allowed,
-`tsh` will use it as the default value for the corresponding flag.
-Prior versions of Teleport would return an error from `tsh db` commands if those
-flags were empty (the only exception is Redis, where it would choose the Redis
-username "default" for `--db-user`).
+#### SSH node open dial no longer supported
 
-#### Auto-Discovered Database Names
+Teleport 14 no longer allows connecting to OpenSSH servers not registered with
+the cluster. Follow the updated agentless OpenSSH integration [guide](docs/pages/server-access/guides/openssh.mdx)
+to register your OpenSSH nodes in the cluster’s inventory.
 
-In Teleport 14, database discovery via `db_service` config will enforce the same
-name validation that we enforce for databases created via `tctl`, static config,
-and `discovery_service`.
+You can set `TELEPORT_UNSTABLE_UNLISTED_AGENT_DIALING=yes` environment variable
+on Teleport proxy to temporarily re-enable the open dial functionality. The
+environment variable will be removed in Teleport 15.
 
-The regex used to constrain database names is: `^[a-zA-Z]([-a-zA-Z0-9]*[a-zA-Z0-9])?$`
+#### Proxy protocol default change
 
-So databases in AWS and Azure must:
-1. start with a letter (uppercase and lowercase are both allowed).
-2. contain only letters, digits, and hyphens.
-3. end with a letter or digit (no trailing hyphens).
+Starting from version 14, Teleport will require users to explicitly enable or
+disable PROXY protocol in their `proxy_service`/`auth_service` configuration
+using `proxy_protocol: on|off` option.
 
-#### MongoDB Server versions prior to 3.6 are no longer supported
+Users who run their proxies behind L4 load balancers with PROXY protocol
+enabled, should set `proxy_protocol: on`.  Users who don’t run Teleport behind
+PROXY protocol enabled load balancers, should disable `proxy_protocol: off`
+explicitly for security reasons.
 
-Teleport 14 includes an update to the MongoDB driver. Due to the MongoDB team
-dropping support for Servers prior to version 3.6, Teleport also will no longer
-be able to support these old server versions.
+By default, Teleport will accept the PROXY line but will prevent connections
+with IP pinning enabled. IP pinning users will need to explicitly enable/disable
+proxy protocol like explained above.
 
-#### DynamoDB Billing Mode
+See more details in our [documentation](docs/pages/management/security/proxy-protocol.mdx).
+
+#### Legacy deb/rpm package repositories are deprecated
+
+Starting from the next major release Teleport 15, legacy package repositories
```

Comment:
I would say something like:

Teleport 14 will be the last release published to the legacy package repositories at deb.releases.teleport.dev and rpm.releases.teleport.dev. Starting with Teleport 15, packages will only be published to the new repositories at apt.releases.teleport.dev and yum.releases.teleport.dev.